serve.py

```diff
@@ -174,6 +177,44 @@ async def server_info(request):
     )
 
 
+async def _fetch(url):
+    querystring = {"page_size": "100"}
+    headers = {"Accept": "application/json", "Content-Type": "application/json"}
+    resp = requests.request("GET", url, headers=headers, params=querystring).json()
+    results = resp["results"]
+    while resp["next"]:
+        resp = requests.request("GET", resp["next"], headers=headers).json()
+        results.extend(resp["results"])
+    assert len(results) == resp["count"]
```

Comment:
I wonder if it would be better to not completely crash here but return what we got but somehow report the error to us to fix? Do you get/monitor logs from this redirector service?

Reply:
removing it for now. we probably should refactor this service a bit.